cms/static/cms/js/modules/cms.page_type_select.js

```diff
@@ -0,0 +1,26 @@
+//##############################################################################
```

Comment:
I would say we name it ``cms.wizzards.js`` and create and encapsulate all js related functions within the wizard (there more in the html files)